<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>场集工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>工作目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenDir">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLineEdit" name="dirEdit">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dirButton">
       <property name="text">
        <string>浏览</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>工程名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="epEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>场</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>工具</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QWidget" name="toolBoxPage1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>207</width>
            <height>292</height>
           </rect>
          </property>
          <attribute name="label">
           <string>拼色板</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>色板素材文件夹</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="csheetFFNameEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>色板名称前缀</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="csheetPrefixEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>色板底板</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="backDropBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>色板名称</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="csheetNameEdit">
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>色板上传至</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="csheetDestEdit">
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="csheetOpenCheck">
             <property name="text">
              <string>创建后打开色板</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="csheetUpCheck">
             <property name="text">
              <string>创建后上传色板</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="sheetButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>创建色板</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>打开色板</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder></zorder>
          <zorder>sheetButton</zorder>
          <zorder>openButton</zorder>
          <zorder>csheetOpenCheck</zorder>
          <zorder>csheetUpCheck</zorder>
          <zorder>verticalSpacer_3</zorder>
         </widget>
         <widget class="QWidget" name="toolBoxPage2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>211</width>
            <height>292</height>
           </rect>
          </property>
          <attribute name="label">
           <string>上传下载</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="1">
              <widget class="QLineEdit" name="imageFNameEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="videoFNameEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>视频文件夹名称</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>单帧文件夹名称</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>视频上传至</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="videoDestEdit">
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>单帧上传至</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="imageDestEdit">
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>上传</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="videoUpCheck">
               <property name="text">
                <string>视频</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="imageUpCheck">
               <property name="text">
                <string>单帧</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>下载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="videoDownCheck">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>视频</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="imageDownCheck">
               <property name="text">
                <string>单帧</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="syncButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>上传及下载</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>高级设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>服务器路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonOpenServer">
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="serverEdit">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="serverButton">
           <property name="text">
            <string>浏览</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="videoFolderEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>服务器视频路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="imageFolderEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>服务器单帧文件夹</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nuke路径</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="nukeEdit">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="nukeButton">
           <property name="text">
            <string>浏览</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>184</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSheet">
   <property name="text">
    <string>sheet</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>sync</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>server</string>
   </property>
  </action>
  <action name="actionDir">
   <property name="text">
    <string>dir</string>
   </property>
  </action>
  <action name="actionNuke">
   <property name="text">
    <string>nuke</string>
   </property>
  </action>
  <action name="actionUpdateUI">
   <property name="text">
    <string>updateUI</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>openDir</string>
   </property>
  </action>
  <action name="actionOpenServer">
   <property name="text">
    <string>openServer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dirButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nukeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionNuke</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverButton</sender>
   <signal>clicked()</signal>
   <receiver>actionServer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSync</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sheetButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSheet</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenDir</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenServer</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenServer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
